### PR TITLE
Manual backport #877

### PR DESCRIPTION
**Pull request by Fenn-CS**
Backport https://github.com/nextcloud/logreader/pull/877

### CODE REVIEW

File: .github/workflows/phpunit-oci.yml [hunk 1]
```diff
@@ -38,7 +38,7 @@ jobs:
 
     strategy:
       matrix:
-        php-versions: ['8.0']
+        php-versions: ['8.0', '8.1', '8.2']
```
Comment (nickvergessen) [inline .github/workflows/phpunit-oci.yml:41]:
```suggestion
        php-versions: ['8.0']
```

File: .github/workflows/phpunit-pgsql.yml [hunk 2]
```diff
@@ -38,7 +38,7 @@ jobs:
 
     strategy:
       matrix:
-        php-versions: ['8.0']
+        php-versions: ['8.0', '8.1', '8.2']
```
Comment (nickvergessen) [inline .github/workflows/phpunit-pgsql.yml:41]:
```suggestion
        php-versions: ['8.0']
```

File: .github/workflows/phpunit-sqlite.yml [hunk 3]
```diff
@@ -38,7 +38,7 @@ jobs:
 
     strategy:
       matrix:
-        php-versions: ['8.0']
+        php-versions: ['8.0', '8.1', '8.2']
```
Comment (nickvergessen) [inline .github/workflows/phpunit-sqlite.yml:41]:
```suggestion
        php-versions: ['8.0']
```

**Review by nickvergessen — COMMENTED** [review]
It's enough to test on one database version against the PHP version, so save CI time and our planet

**Review by nickvergessen — APPROVED** [review]

**Comment by Fenn-CS** [comment]
Closes : https://github.com/nextcloud/logreader/issues/868

**Pull request merged**